components/metrics/lib.rs

```diff
@@ -33,7 +33,7 @@ pub trait ProgressiveWebMetric {
 /// maximum task time is 50ms (in ns)
 pub const MAX_TASK_NS: u64 = 50000000;
 /// 10 second window (in ns)
-const INTERACTIVE_WINDOW_SECONDS_IN_NS: u64 = 10000000000;
+const INTERACTIVE_WINDOW_SECONDS_IN_NS: Duration = Duration::from_nanos(10000000000);
```

Comment:
I think the below will make this a bit more readable:

```suggestion
const INTERACTIVE_WINDOW_DURATION: Duration = Duration::from_secs(10);
```